src/XMakeBuildEngine/Construction/ProjectImportElement.cs

```diff
@@ -44,26 +44,36 @@ public string Project
             get
             {
                 return
-                    FileUtilities.FixFilePath(
-                        ProjectXmlUtilities.GetAttributeValue(XmlElement, XMakeAttributes.project));
+                    FileUtilities.FixFilePath(ProjectXmlUtilities.GetAttributeValue(XmlElement, XMakeAttributes.project));
             }
 
             set
             {
                 ErrorUtilities.VerifyThrowArgumentLength(value, XMakeAttributes.project);
+
                 ProjectXmlUtilities.SetOrRemoveAttribute(XmlElement, XMakeAttributes.project, value);
                 MarkDirty("Set Import Project {0}", value);
             }
         }
 
         /// <summary>
-        /// Location of the project attribute
+        /// Gets the Implicit state of the element: true if the element was not in the read XML.
         /// </summary>
-        public ElementLocation ProjectLocation
+        // TODO: *should* this be public? if it's not, you can't determine if an import is implicit from the public OM.
+        public bool Implicit
         {
-            get { return XmlElement.GetAttributeLocation(XMakeAttributes.project); }
+            get { return XmlElement.HasAttribute(XMakeAttributes.@implicit); }
```

Comment:
Could we represent whether an import is implicit or not with a field in this class instead of a separate attribute?  What happens if someone actually uses the attribute?

```xml
<Import Project="foo.targets" _Implicit="Foo" />
```